docs/user-guide/services.md

```diff
@@ -1,6 +1,49 @@
 # Services
 
-## Running the monitor data service
+## Overview
+
+### Main services
+
+The following main services are available:
+
+```sh
+python -m beamlime.services.monitor_data --instrument dummy
+python -m beamlime.services.detector_data --instrument dummy
+python -m beamlime.services.data_reduction --instrument dummy
+python -m beamlime.services.timeseries --instrument dummy
```

Comment:
(Not related to this PR) 
Should we expose them as commands so that we can type `beamlime` and then auto complete service names?